src/worker.py

```
@@ -1197,6 +1387,83 @@ async def api_create_session(req, env):
+            "DELETE FROM sessions WHERE activity_id=?"
+        ).bind(act_id).run()
+        await env.DB.prepare(
+            "DELETE FROM enrollments WHERE activity_id=?"
+        ).bind(act_id).run()
+        await env.DB.prepare(
+            "DELETE FROM activity_tags WHERE activity_id=?"
+        ).bind(act_id).run()
+        await env.DB.prepare(
+            "DELETE FROM activities WHERE id=? AND host_id=?"
+        ).bind(act_id, user["id"]).run()
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)